keyboards/AT101_Blackheart/config.h

```diff
@@ -0,0 +1,46 @@
+#ifndef CONFIG_H
```

Comment:
Could you change the top two lines to just `#pragma once`, and remove the endif at the end.

Reply:
Fixed.